packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -183,6 +183,33 @@ enum SchedulerPhase {
   postFrameCallbacks,
 }
 
+/// This callback invoked when a request for [DartPerformanceMode] is disposed.
+///
+/// See: [PerformanceModeRequestHandle].
```

Comment:
Please follow our doc convention for "see also" sections. Search for examples in the repo.